lib/livebook/application.ex

```diff
@@ -69,7 +69,7 @@ defmodule Livebook.Application do
 
   defp display_startup_info() do
     if Phoenix.Endpoint.server?(:livebook, LivebookWeb.Endpoint) do
-      IO.ANSI.format([:blue, "Livebook running at #{access_url()}"]) |> IO.puts()
+      IO.puts("[Livebook] Application running at #{access_url()}")
```

Comment:
Blue is hard to read on black background, so I went with no color and a clear prefix.

Reply:
With log level `:notice` this sounds good to me, it was colored only to stand out from all the other text :>